graphistry/feature_utils.py

```diff
@@ -47,6 +47,16 @@
         SuperVectorizer = Any
         GapEncoder = Any
         SimilarityEncoder = Any
+    try:
+        from cuCat import (
+            SuperVectorizer,
+            GapEncoder,
+            SimilarityEncoder,
+        )
+    except:
+        SuperVectorizer = Any
+        GapEncoder = Any
+        SimilarityEncoder = Any
```

Comment:
this is a bit wrong, right? depending on if someone wants to use dirty_cat vs cuCat, which is a runtime engine decision, this would statically override at load-time, and not allow on-the-fly switching, right?